include/unifex/task.hpp

```diff
@@ -455,7 +473,7 @@ struct _sr_thunk_promise final {
    */
   struct type final
     : _sr_thunk_promise_base
-    , _return_value_or_void<T>::type {
+    , _return_value_or_void<T, false>::type {
```

Comment:
Was this a TODO item?  I can't remember.  We can land like this, but we should probably follow up later with some analysis to determine whether `_sr_thunk_task` should have a no-throw version.

Reply:
I think we just missed this. I'll add a todo here. Most of the methods here are already noexcept, but this is dependent on _sr_thunk_promise_base which has an await_transform that we also were not sure if it should be noexcept yet.